src/server/status/index.js

```diff
@@ -29,7 +34,8 @@ export default function (kbnServer, server, config) {
           build_snapshot: matchSnapshot.test(config.get('pkg.version'))
         },
         status: kbnServer.status.toJSON(),
-        metrics: kbnServer.metrics
+        metrics: kbnServer.metrics,
+        stats,
```

Comment:
> Continuing to put it on the status api and migrate as a second step, or start with a new api for these stats?

Since we have identified ways that the existing status API should be split up and renamed to match the conventions of Elasticsearch (although having the `/api` prefix makes sense for Kibana), I would suggest continue to put these stats in the current status API

> we could also cleanup the current API in that new API in terms of what it returns as it's a mix of "info" and "stats" even though it purports to be "status" only.

Cleaning up the existing API is important. Let's open another issue for that, and figure out what needs to be split, and what the different endpoints should be named.